src/general/StringUtils.cs

```diff
@@ -41,4 +41,15 @@ public static string FormatNumber(this double number, bool withSuffix = true)
 
         return number.ToString("0.#", CultureInfo.CurrentCulture);
     }
+
+    public static string FormatDecimal(this double number, int decimalPlaces)
+    {
+        var format = "0.";
+        for (var i = 0; i < decimalPlaces; i++)
+        {
+            format += "0";
```

Comment:
The problem isn't chopping off extra digits, it's always printing the ".0" at the end. If you have a more elegant way to generate this format string I'm all ears.

Reply:
I think that maybe this method should *not* exist. It's hard to avoid a string object creation without a lookup table.
`F2` is a format string used elsewhere in the code base already. I just don't see how this general helper method could be made.